src/assets/specs/command-api.yaml

```
@@ -0,0 +1,109 @@
+openapi: "3.0.0"
+info:
+  title: "Rancher Desktop API"
+  version: 0.0.1
+paths:
+  /v0/list-settings:
```

```suggestion
  /v0/settings:
```

Usually, a verb should not be included in the resource path.

---

This PR is still using the prototype API; redesigning the API will be part of #1925.

But otherwise, I agree, that there should be a single "settings" endpoint, that you can fetch via GET and update via PUSH.

---

Agreeing with both of you -- the endpoints should be `GET /VERSION/settings` and `PUT /VERSION/settings`, but not for this PR.